### DISCUSSION TITLE
Cannot creat a rule for a transaction for its amount to be added to a piggy bank

### DISCUSSION THREAD

**Discussion by gregblink** [firefly-iii/firefly-iii#4405] [Firefly III] [answered]
Couldn't reproduce it in demo site unfortunately, so the problem obviously comes from my end.

I stripped the transaction and the piggy bank out of their fanciness, they're bare naked, there's no categories, none belongs to a group. It simply won't do, and i dunno why. The exact same thing works flawlessly in the demo site.
Here after a screencap of the error message.
![image](https://user-images.githubusercontent.com/79115175/108084464-1d94c500-706c-11eb-80bd-5fcacdfb78da.png)

**Comment by JC5** [comment, marked as answer]
I know, it's a bug :(

The ticket has been closed, sorry about that. You weren't able to find it I guess. But it's duly noted and a fix will be part of the new release.

https://github.com/firefly-iii/firefly-iii/issues/4133
